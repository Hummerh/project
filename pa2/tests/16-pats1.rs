fn main() {
      let x : () = ();

      match (()) {
            () => { (); }
      }
}
