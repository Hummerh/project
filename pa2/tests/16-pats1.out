(crate
    (items
        (fn-def
            (id
                (main))
            (block
                (let
                    (pat-id
                        (id
                            (x)))
                    (type-unit)
                    (unit))
                (match
                    (unit)
                    (match-arms
                        (match-arm
                            (pats
                                (pat-unit))
                            (block
                                (unit)))))))))